osu.Game/Overlays/BeatmapSet/AuthorInfo.cs

```diff
@@ -126,6 +127,25 @@ public Field(string first, string second, FontUsage secondFont)
                     },
                 };
             }
+
+            public Field(string first, DateTimeOffset second, FontUsage secondFont)
+            {
+                AutoSizeAxes = Axes.Both;
+                Direction = FillDirection.Horizontal;
+
+                Children = new[]
+                {
+                    new OsuSpriteText
+                    {
+                        Text = $"{first} ",
+                        Font = OsuFont.GetFont(size: 13)
+                    },
+                    new DrawableDate(second)
+                    {
+                        Font = secondFont.With(size: 13)
+                    }
```

Comment:
The appearance of these elements doesn't match web design.

Reply:
I'm not quite sure what's wrong here. Can you elaborate?

Reply:
OK, I highlighted the wrong place, but what I meant to point out is the difference in typography - web tooltip has date in white, bold font, time + offset in regular, slightly dimmed colour, while lazer-side it's all regular white text.

Reply:
This would require a framework change for how tooltips work which is beyond the scope of this PR.

Reply:
I don't think that's correct. Custom tooltips are already supported - check the `IHasCustomTooltip` interface and its lazer-side implementations.

Now changing `DrawableDate` to use it *might* warrant a separate PR, but I'm not overly convinced it does.

Reply:
You're right. I switched to `IHasCustomTooltip` now.